building/Windows.md

```diff
@@ -0,0 +1,133 @@
+# Windows
+
+If we want we can build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows with Cygwin and WSL.
+
+## Cygwin
+
+In the beginning, we need to download libUSB and set the QEMU environment.
+
+To do that we need to simply download QEMU from the official website which is targeted to our system. The same
+situation is with libUSB, but we need to follow the instructions provided by the creator.

```

Comment:
I would simplify it, no need to tell 3 times that something needs to be downloaded.